commands/system/vpn/vpn-start.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Connect
+# @raycast.mode compact
+
+# Optional parameters:
+# @raycast.icon 📡
+
+# @Documentation:
+# @raycast.packageName VPN
+# @raycast.description Start VPN connection.
+# @raycast.author Alexandru Turcanu
+# @raycast.authorURL https://github.com/Pondorasti
+
+
```

Comment:
@unnamedd  how do you plan to make the `vpn-config.sh` discoverable, hence it will not show up in the big `README.md`? I could write a small text at the top of each file, similar to this one:

```suggestion
Requirements: download and configure `vpn-config.sh`
Afterward, place it in the same directory
```

Reply:
There’s no need to make it discoverable in the README or even by Raycast, it will be used only by the files you wrote, the most important information to be presented in the documentation are already in the .sh files. Also, it will be downloaded with your file as well, because today, when you download the Script Commands today, you will automatically download the config file. Don’t worry about the config file. It is there just to reduce duplication of code and place where the user will configure the VPN name 😃. 

In the future, when a kind of Script Command Manager we might have a list of files related to that group of files (like here).